.github/workflows/update-discourse-data.yml

```
@@ -0,0 +1,48 @@
+name: Update Discourse data
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 6 * * *"
+
+jobs:
+  update-discourse-data:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+
+    steps:
+      - uses: actions/create-github-app-token@v1
+        id: app-token
+        with:
+          app-id: ${{ vars.WEBSITE_UPDATER_APPID }}
+          private-key: ${{ secrets.WEBSITE_UPDATER_KEY }}
+
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          token: ${{ steps.app-token.outputs.token }}
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.x"
```

In CI workflows, I would rather pin the Python version, to avoid surprises.
There is also already a newer version of this Action.
How about:


```suggestion
      - name: Set up Python
        uses: actions/setup-python@v6
        with:
          python-version: "3.14"
```